2023 SC Election Kickoff Files

geekygirldawn opened the pull request:
Adding the initial files required to run the 2023 Steering Committee election.
 
Before this can be merged, we need the following approvals:
* @knative/steering-committee - confirm that details are correct and acceptable to SC (number of seats, schedule, etc.)
* @knative/steering-committee - approve / nominate @ReToCode and me as election officers for this election
* @jberkus - confirm that I haven't mangled any of the things needed by Elekto, and confirm that you are available on the key dates if there are any Elekto problems. Note: I've left you on the elections email alias and in the owners file in case anyone reports Elekto issues via email or you need to merge any urgent changes due to issues with Elekto.

@aliok - while you aren't an election officer this round, you are good at spotting mistakes, so if you have time, I'd appreciate a review :) 

/hold

aliok commented on elections/2023-SC/README.md:
```diff
@@ -0,0 +1,163 @@
+
+**Nomination**
+
+If you want to stand for election for a Contributor Seat, open a PR against the
```
```suggestion
If you want to stand for election for a Steering Committee Seat, open a PR against the
```

aliok commented on elections/2023-SC/README.md:
```diff
@@ -0,0 +1,163 @@
+
+You can find [a sample template in the folder](./nomination-template.md).
+
+Once you have created the PR, you may email knative-dev and/or knative-users
```
```suggestion
Once you have created the PR, you may email knative-dev@googlegroups.com and/or knative-users@googlegroups.com
```

aliok commented on elections/2023-SC/README.md:
```diff
@@ -0,0 +1,163 @@
+
+Once you have created the PR, you may email knative-dev and/or knative-users
+*once* to let people know about your candidacy and encourage endorsements as
+comments on the PR. Please use something like this for your email in order to
```
```suggestion
comments on the PR. Please use a text similar to the text below for your email in order to
```

aliok commented on elections/2023-SC/README.md:
```diff
@@ -0,0 +1,163 @@
+
+## Voting Process
+
+Elections will be held using [Elekto], an online voting tool created by CNCF 
+intern Manish Sahani.  Elekto has one critical advantage over CIVS, which is
+that it relies on GitHub Oauth to log you in to vote, instead of relying on 
+email. More details on voting are in the [Elekto documentation].
```
```suggestion
Elections will be held using [Elekto], an online voting tool created
as a CNCF internal project. As a reminder it relies on GitHub Oauth to log you
in to vote, instead of relying on email. More details on voting are in the [Elekto documentation].
```

aliok commented on elections/2023-SC/README.md:
```diff
@@ -0,0 +1,163 @@
+email. More details on voting are in the [Elekto documentation].
+
+Thus, when you go to [Elekto] you will be prompted to log in your GitHub account.
+Please do, so, and then click on "Explore Election" to look at the list of 
```
```suggestion
Please do so, and then click on "Explore Election" to look at the list of 
```

aliok commented on elections/2023-SC/README.md:
```diff
@@ -0,0 +1,163 @@
+anyone except the voter, or in aggregate form.
+
+### Decision
+
+The newly elected body will be announced on the knative-dev mailing list.
```
```suggestion
Ballots are compiled by Elekto and all candidates are ranked using the [Condorcet]
method, Schultze variant.

The newly elected body will be announced on the knative-dev mailing list.
```

aliok commented on elections/2023-SC/README.md:
```diff
@@ -0,0 +1,163 @@
+[Eligibility for candidacy]: https://github.com/knative/community/blob/main/mechanics/SC.md#candidate-eligibility
+[Eligibility for voting]: https://github.com/knative/community/blob/main/mechanics/SC.md#candidate-eligibility#voter-eligibility
+
+[file a voting exception]: https://elections.knative.dev/app/elections/2023-SC/exception
```
```suggestion
[file a voting exception]: https://elections.knative.dev/app/elections/2023-SC/exception
[Condorcet]: https://en.wikipedia.org/wiki/Condorcet_method
```

geekygirldawn commented:
@aliok Thank you so much! I used the last SC election as a template and forgot how many improvements were made in the more recent TOC election :)

jberkus commented:
Looks good.  We'll need to check the GH user names for case-sensitivity though; that's still not fixed.

lance commented:
/lgtm
Thanks for doing this @geekygirldawn. Thumbs up to @ReToCode.

geekygirldawn commented:
I’ll push a fix this weekend to fix the case issues. I have a script for that already 😁

geekygirldawn commented:
/hold cancel

I addressed the feedback from @aliok and updated voters.yaml as @jberkus suggested to get around the case sensitivity bug in Elekto, so I think this is ready for @knative/steering-committee approval and merge.

geekygirldawn commented:
> I cannot see voters.yaml. But didn't we lower the requirements for voters?

We talked about it, but I don't think we ever made it official, since if so, we should have updated it here: https://github.com/knative/community/blob/main/mechanics/SC.md#voter-eligibility 

I think we can be generous with exception approvals and then if we want to lower the bar, we can make it official after this election?

nainaz commented:
Since election process haven't kick started yet. I will vote for doing it before. 
Filing an exception process is that extra step that some/most folks won't take. 
I think we should lower it to 30 or 25.

geekygirldawn commented:
> Since election process haven't kick started yet. I will vote for doing it before. Filing an exception process is that extra step that some/most folks won't take. I think we should lower it to 30 or 25.

I would need to re-run the data gathering (not a big deal), so these numbers aren't exact, but right now, we have 55 people in voters.yaml using the devstats 50+ contributions criteria. If we lowered it to 25 contributions, we would get an additional ~26 people.

Can SC make this decision today to give me time to update the voters.yaml file before we kick off the election tomorrow?

nainaz commented:
Steering Committee confirms that we want to lower the contributions to 25 for voters.

geekygirldawn commented:
> Steering Committee confirms that we want to lower the contributions to 25 for voters.

I've added a commit with the additional voters in voters.yaml. Is someone on steering planning to update https://github.com/knative/community/blob/main/mechanics/SC.md#voter-eligibility or should open a second PR to make that update?

geekygirldawn commented:
@lance @csantanapr @nainaz or other @knative/steering-committee members. Now that #1452 is merged, this is ready to be merged.

lance commented:
/lgtm

csantanapr commented:
/lgtm
/approve
